presto-main/src/main/java/com/facebook/presto/execution/AddColumnTask.java

```diff
@@ -95,7 +96,11 @@ public String getName()
                 metadata,
                 parameters);
 
-        ColumnMetadata column = new ColumnMetadata(element.getName().getValue(), type, element.getComment().orElse(null), null, false, columnProperties);
+        Optional<Object> maybeDefaultValue = element.getDefaultValue()
```

Comment:
`s/maybeDefaultValue/defaultValue` (by convention, the `Optional` type conveys the "maybe-ness")